python/cuml/benchmark/runners.py

```diff
@@ -166,8 +173,10 @@ def _run_one_size(
 
         cpu_accuracy = 0.0
         if run_cpu and algo_pair.cpu_class is not None:
+            algo_pair.setup_cpu(data, **param_overrides)
```

Comment:
How about returning the setup results here as a local and passing them in to run_cpu?
The setup_cpu function could return a dict and the bench_func could take them as **kwargs which will be empty in most cases but non-empty for functions that need it?

Reply:
Hi John, in the case of FIL, the setup result is actually a cuml_obj which we can directly call `predict` method on this object. I am not sure why this could be converted to a dict?

Reply:
Hi John, I updated the code based on your comments. Can you take a quick look? Thanks!

Reply:
Looks good now!